bigquery_etl/glam/templates/probe_counts_v1.sql

```diff
@@ -106,9 +106,11 @@ WITH probe_counts AS (
     {{ aggregate_grouping }}
 )
 
-{% if channel == "release" %}
+{% if channel == "release" and is_scalar %}
 ,
   -- Fix All OS client counts which were originally calculated taking only 10% of Windows due to sampling.
+  -- This is only relevant for scalars, as histograms are already made right during client normalization.
```

Comment:
```suggestion
 -- This is only relevant for scalars, as histograms are already normalized during client normalization.
```
maybe this wording is slightly better?